src/styles/nav.scss

```diff
@@ -0,0 +1,50 @@
+.nav {
+  display: flex;
+
+  &__list {
+    margin: 0;
+    padding: 0;
+    display: flex;
+    list-style: none;
+    text-transform: uppercase;
+  }
+
+  &__list-el {
+    display: flex;
+
+    &:not(:first-child) {
+      margin-left: 20px;
+    }
+  }
+
+  &__list-link {
+    line-height: 60px;
+    text-decoration: none;
+    color: var(--main-letters-color);
+    position: relative;
+    font-weight: 500;
+    transition-duration: 300ms;
+
+    &:hover {
+      transition-duration: 300ms;
```

Comment:
The transition-duration property is repeated in the :hover state, which is redundant because it's already defined for the element. Remove the transition-duration from the :hover state.